tcb/.conf/update.json

```diff
@@ -1,2 +1,10 @@
 [
+    {
```

Comment:
If the tcb template does not have this files, should it be here?

Reply:
Is not that it does not have it, but is not the application one. These are the specific TCB ones, they are different.